pkg/apis/operator/v1alpha1/metaoperatorconfig_types.go

```diff
@@ -53,7 +54,7 @@ type MetaOperatorConfigStatus struct {
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
 	// ServiceStatus defines all the status of a operator
 	// +optional
-	ServiceStatus map[string]*CrStatus `json:"serviceStatus,omitempty"`
+	ServiceStatus map[string]CrStatus `json:"serviceStatus,omitempty"`
```

Comment:
can you manually verify if this change can work well with the current CSV that published to quay.io?

Reply:
I verified it with `operator-sdk run --local` to test the function.